framework/tests/src/tests/WholeProgramInferenceValidationTest.java

```
@@ -19,6 +19,13 @@ public WholeProgramInferenceValidationTest(List<File> testFiles) {
         super(testFiles, WholeProgramInferenceTestChecker.class, "value", "-Anomsgtext");
     }
 
+    @Override
+    public void run() {
+        if (new File("tests/whole-program-inference/annotated/").exists()) {
+            super.run();
+        }
```

Should this raise a user-friendly error if the directory is not found?